etl/resources/output_config.py

```diff
@@ -61,6 +67,12 @@ def wikipedia_anti_recommendations_file_path(self) -> Path:
                 / "wikipedia_anti_recommendations.jsonl"
             )
 
+        @property
+        def wikipedia_arkg_file_path(self) -> Path:
+            """The Path of the file that contains a Wikipedia ARKG."""
+
+            return self.anti_recommendations_directory_path / "wikipedia_arkg.ttl"
```

Comment:
This should be a stem if the output RDF format is configurable.

Reply:
See the table here:

https://www.npmjs.com/package/rdf-parse

Reply:
I changed the file path to have a configurable format.